docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,99 @@
+
+            - convert the chiller.capacity_validation_points to a dictionary with the keys being a list of two values - [chilled_water_supply_temperature, condenser_temperature], and values being the capacity_validation_point itself.  Sorting the validation points will allow the calculations that follow to be much simpler.  Start by creating a dict capacity_validation_pts_dict: `capacity_validation_pts_dict = {}`
+            - look at each value in chiller.capacity_validation_points: `for capacity_validation_point in chiller.capacity_validation_points:`
+                - add the point to capacity_validation_pts_dict: `capacity_validation_pts_dict[[capacity_validation_point.chilled_water_supply_temperature, capacity_validation_point.condenser_temperature]] = capacity_validation_point.result`
+            
+            - do the same conversion with the chiller.power_validation_points and storing the results in power_validation_pts_dict.  However in this case, the values are a list of points because we expect different loads at each point: `power_validation_pts_dict = {}`
+            - look at each value in chiller.power_validation_points: `for power_validation_point in chiller.power_validation_points:`
+                - set the default to a blank list: `power_validation_pts_dict.setdefault([[power_validation_point.chilled_water_supply_temperature, power_validation_point.condenser_temperature]], [])`
+                - append the power validation point to the list at this given: `power_validation_pts_dict[[power_validation_point.chilled_water_supply_temperature, power_validation_point.condenser_temperature]].append(power_validation_point.result)`
+
+            - create a dictionary of the expected capacities for use in the power validation check: `given_capacities = {}`
+            - the following lines or logic do the capacity validation check:
+            - create a list of non-matching capacity validation points: `non_matching_capacity_validation_points = []`
+            - create a list of missing capacity validation points: `missing_capacity_validation_points = []`
+            - loop through the expected_chwt_temps: `for cwt in expected_cwt_temps:`
+                - loop through each expected_ecwt_temps: `for ect in expected_ecwt_temps:`
+                    - look for the capacity validation point in capacity_validation_pts_dict: `if capacity_validation_pts_dict[[cwt, ect]]:`
+                        - calculate the expected capacity by multiplying the result of the formula by the rated_capacity: `expected_capacity = (cap_f_t_coefficients[0] + cap_f_t_coefficients[1] * cwt + cap_f_t_coefficients[2] * cwt^2 + cap_f_t_coefficients[3] * ect + cap_f_t_coefficients[4] * ect^2 + cap_f_t_coefficients[5] * cwt * ect) * rated_capacity`
```

Comment:
This formula looks correct cut recommend upsating ect and cwt nomenclature

Reply:
everything has been updated to be ect and cwt

Reply:
The typical abbreviation for chilled water is CHW so I recommend that any variable that pertains to chilled water use CHW and any variable that represents condensing water be CW. Looks like they both use CW regardless of whether they are chilled or condenser water. This is a suggestion - feel free to ignore but I found it was a little confusing with CW for both. Maybe use the nomenclature in 90.1 with CHWT and ECT.